dynamic-layers/qt5/recipes-updatehub/agent-sdk-qt/agent-sdk-qt_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "agent-sdk-qt"
+DEPENDS = ""
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=838c366f69b72c5df05c96dff79b35f2"
+
+SRCREV = "819e52713154fa1c3eea7550be1395411602d155"
+SRC_URI = "git://github.com/updatehub/agent-sdk-qt.git"
+
+S = "${WORKDIR}/git"
+
+inherit qmake5
+
+PACKAGECONFIG ??= "qmlplugin"
+PACKAGECONFIG[qmlplugin] = "CONFIG+=qmlplugin,,qtdeclarative,qtdeclarative-qmlplugins"
+
+EXTRA_QMAKEVARS_PRE += " \
+  PREFIX=${prefix} \
+  ${PACKAGECONFIG_CONFARGS} \
+"
+
+PACKAGES += "${PN}-qmlplugin"
+
+FILES_${PN}-qmlplugin += "${OE_QMAKE_PATH_QML}/updatehub"
```

Comment:
You should rename the recipe to updatehub-agent-sdk-qt as we need to have the updatehub namespace in all recipes.

After the FILES, add:

```
RDEPENDS_${PN}-qmlplugin += "qtdeclarative-qmlplugins"
```

and you should drop the last argument in `PACKAGECONFIG` line. This is because the rdepends must be added to this specific package, not the main one.